redash/utils/__init__.py

```diff
@@ -63,7 +63,6 @@ def gen_query_hash(sql):
         2. SELECT 1 FROM table where column='value';
     """
     sql = COMMENTS_REGEX.sub("", sql)
-    sql = "".join(sql.split()).lower()
```

Comment:
1. We need to keep the join/split behavior as this takes care of removing multiple whitespaces. 
2. Need to update the docstring to actually reflect reality :)

Reply:
@arikfr Thanks for catching that. I shall perform an update to correct these